lib/erl_interface/src/Makefile.in

```diff
@@ -668,7 +672,18 @@ EXTRA = \
 	README.internal \
 	$(TARGET)/eidefs.mk
 
-release: opt
+PKG-CONFIG = \
+	$(TARGET)/erl_ei.pc
+
+ifneq ($(DESTDIR),)
```

Comment:
When calling `./otp_build release [-a] <PATH>` or `make release RELEASE_ROOT=<PATH>` the DESTDIR is not defined, thus the pkg-config rule will not be called.

Reply:
Yes, but `DESTDIR` is not necessarily used when installing with `make install`, so now `erl_ei.pc` will be lost when not using `DESTDIR`.

Reply:
Ok. My next version will be like this:

```
./configure --enable-pkg-config

make install DESTDIR=<PATH>

or 

make install
```

Both will install pkg-config .pc file because the user has enabled pkg-config. However for `otp_build release` and `make release` cases, it will not take effect if the user hasn't enabled the pkg-config.